actionpack/test/controller/http_token_authentication_test.rb

```diff
@@ -79,12 +79,12 @@ def authenticate_long_credentials
     end
   end
 
-  test "authentication request with badly formatted header" do
-    @request.env['HTTP_AUTHORIZATION'] = "Token foobar"
+  test "authentication request with text" do
```

Comment:
My impression was that the existing test was testing for a badly formatted header to fail authentication, when in reality the "badly formatted header" in question is actually valid according to the spec, so the existing (invalid) test was removed, and a new one was added to ensure authorization headers without key/value pairs are acceptable.